server/models/Appointment.js

```diff
@@ -0,0 +1,28 @@
+const mongoose = require('mongoose');
+
+const { Schema } = mongoose;
+const Appointment require('./Appointment');
+const Pet require('./Pet');
+const Service require('./Service');
+
+const appointmentSchema = new Schema({
+  date: {
+    type: Integer,
```

Comment:
Why are we declaring date as an integer? It should be string or date

Reply:
For date, I think we should use the format numeric format that you get while using Date.now() (1628868365654 - for example) and convert it to the XX/XX/XXXX format for display, the reason for that is that Graphql does not have a default date data type, and we would need to set a custom type for that (scalar type) which I am not familiar with yet, I want to ask Nelio/Fulton about it, but for the time being I put it as an integer. Hope this makes sense.

Reply:
Since the underlines engine that runs the queries is mongo, it will be much easier to use a String that mongo will convert to date if we need to perform any queries using the date column.

Reply:
Ok, no problem with that, in the resolver I will just create a new instance of date from the date string, which would also work, will do the change.